MaterialLib/MPL/Medium.cpp

```diff
@@ -47,6 +47,11 @@ Property const& Medium::property(PropertyType const& p) const
     return *_properties[p];
 }
 
+bool Medium::isProperty(PropertyType const& p) const
```

Comment:
It's maybe more the question *has* property, isn't it?

Reply:
Yes, maybe `hasProperty` sounds more appropriate. Will change this.